app/tests/services/test_events.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from unittest.mock import patch
+
+from app.services.events import EventsService
+
+
+class TestEventsService(unittest.TestCase):
+
+    def test_is_event_valid(self):
+        valid_event = {"chainId": "123", "type": "transaction"}
+        self.assertTrue(EventsService.is_event_valid(valid_event))
+
+    def test_is_event_invalid(self):
```

Comment:
I think that this test could be a continuation from the previous one. I mean, reading the test function I was thinking wrongly that there was a `is_event_invalid` funtion.

Reply:
Makes sense. Updated [here](https://github.com/safe-global/safe-decoder-service/pull/24/commits/6debdd02f91c2c1d707cb32cad42a75034da3cec)